src/add-search-spec.js

```diff
@@ -144,9 +144,10 @@ describe('Search Results', () => {
 
 function addSearchSpecToProject(argv, config) {
   if (config &&
-    config.appSettings &&
-    config.appSettings.stache &&
-    config.appSettings.stache.search) {
+      config.appSettings &&
+      config.appSettings.stache &&
+      config.appSettings.stache.searchConfig &&
+      config.appSettings.stache.searchConfig.allowSiteToBeSearched) {
```

Comment:
This is a bit of a generic question.  To add searching to a site, is it correct that users were need to add the plugin and add this property to `skyuxconfig.json`?  

If so, I know we could set those defaults in the template they clone, but I'm wondering if we should change the default of `allowSiteToBeSearched` to true?

Reply:
That's a good idea, but where should that default live? Should I assume it to always be true and then check for the false here?